src/histolab/tiler.py

```diff
@@ -337,7 +355,7 @@ def __init__(
         check_tissue: bool = True,
         prefix: str = "",
         suffix: str = ".png",
-        max_iter: int = 1e4,
+        max_iter: float = 1e4,
```

Comment:
Why float? How can you have a float number of iterations?

Reply:
`1e4` in python is float. So we have to decide to change de default value

Reply:
maybe int(1e4) ?

Reply:
as you prefer actually